tests/acceptance/features/coreApiSharePublicLink1/changingPublicLinkShare.feature

```diff
@@ -190,6 +190,8 @@ Feature: changing a public link share
       | ocs_api_version | ocs_status_code |
       | 1               | 100             |
       | 2               | 200             |
+    And the OCS status code should be "104"
+    And the OCS status message should be "user is not allowed to delete the password from the public link"
```

Comment:
This can be removed
```suggestion
```

Reply:
^ 
@2403905 tests are passing. Please, remove passing tests from the expected failure list.